src/validators/issues.rs

```diff
@@ -65,6 +65,8 @@ pub enum IssueType {
     MissingLanguage,
     /// The publisher language code is not valid.
     InvalidLanguage,
+    /// The object has at least one object with the same id.
+    DupplicateObjectId,
```

Comment:
`Duplicate`, with one `p`

Reply:
oups I manage to always make this mistake 🤦‍♂️